apps/ensadmin/src/components/recent-registrations/hooks.ts

```diff
@@ -65,3 +69,86 @@ export function useRecentRegistrations(searchParams: URLSearchParams) {
     },
   });
 }
+
+export function useRecentRegistrationsViaPonder(skipChildrenOfDomains: Array<string> = []) {
```

Comment:
Please note this hook didn't work as expected. It was lacking the`domainOwnerId` value in results, even thought the network response included correct values. It might be something with Ponder Client.